src/zarpaint/napari.yaml

```
@@ -23,6 +23,9 @@ contributions:
   - id: zarpaint.copy_data
     title: Copy Data…
     python_name: zarpaint:copy_data
+  - id: zarpaint.interpolate_widg
+    title: Interpolate
```

```suggestion
    title: Interpolate Slices with Signed Distance Transform
```
?

Maybe we can remove "signed" or `Interpolate Slices (w/ Distance Transform)` if length is an issue